polaris-react/src/components/CalloutCard/CalloutCard.tsx

```diff
@@ -4,6 +4,7 @@ import {CancelSmallMinor} from '@shopify/polaris-icons';
 import {classNames} from '../../utilities/css';
 import type {Action} from '../../types';
 import {Card} from '../Card';
+// eslint-disable-next-line import/no-deprecated
 import {TextContainer} from '../TextContainer';
```

Comment:
Should this get replaced by `AlphaStack`? Or will that happen in another PR?

Reply:
That will happen in the migration PR which @yurm04 is working on